### ISSUE TITLE
Friend Methods Require Implementation When Using Implements Keyword

### ISSUE THREAD

**Issue by jpbro** [WaynePhillipsEA/twinbasic#750]
**Describe the bug**
If you use "Implements" in a class referencing another class that has Friend methods, tB complains about a "missing implementation of member..." for the Friend methods. In VB6 it is not required to implement Friend methods.

**To Reproduce**
Try the following code to see the error reported:

```
Public Class MyClass1
    Friend Property Get TestFriend() As String
        TestFriend = "Doesn't need to be implemented"
    End Property
    
    Public Property Get TestPublic() As String
        TestPublic = "Must be implemented"
    End Property
End Class

Public Class MyClass2
    Implements MyClass1
    
    Public Property Get MyClass1_TestPublic() As String
        MyClass1_TestPublic = "Is Implemented"
    End Property
End Class
```

**Expected behavior**
Perhaps it is good practice to implement Friend methods, but for full compatibility with VB6 I would expect tB to behave like VB6 by not requiring the implementation of Friend methods.

**Desktop (please complete the following information):**
 - OS: Windows 10 
 - twinBASIC compiler version 0.14.31

**Comment by wqweto** [comment]
This must be a self-inflicted problem from an early request to quickly fix `Friend` modifier by implementing it as `Public` until implemented properly.

**Comment by WaynePhillipsEA** [comment]
Hmm yes never noticed this before.  Trying to call a friend method from a class that implements a friendly-interface throws a runtime error: 
<img width="459" alt="tbCannotCallFriendError" src="https://user-images.githubusercontent.com/25738109/155465512-5be24d49-2d8e-40fb-85aa-98e7e906546e.png">

So I'll just match that behaviour in tB by auto-implementing the friend member, throwing this error.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.14.34, thanks!

**Comment by EduardoVB** [comment]
My interpretation is: the interfaces don't have "Friend" members, only Public.

Just the class object can have Friend members, but not the interface taken from it.
